docs/release-notes/delete-namespace-bindings.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+**New Features**
+
+-  CLI: Add a feature where Determined offers the users to delete a workspace namespace binding by
+   using the command ``det w bindings delete <workspace-id> --cluster-name <cluster-name>``. An
+   error will be thrown if the user tries to delete a default binding.
```

Comment:
:orphan:

**New Features**

-  CLI: Add the ability to delete a workspace namespace binding with the command ``det w bindings delete <workspace-id> --cluster-name <cluster-name>``. An
   error will be thrown if attempting to delete a default binding.